js/jquery.fileupload-video.js

```diff
@@ -97,8 +98,11 @@
                 return data;
             },
 
-            // Sets the video element as a property of the file object:
+            // Copy the properties set on the data object to the file object
```

Comment:
The idea for the distinction of `loadVideo` and `setVideo` is that there are cases where you might wanna load the video, but not set it as preview, e.g. for doing client-side processing.